### PR TITLE
Add tests for actuators classes

### PR DESCRIPTION
**Pull request by alessandropalla**

### CODE REVIEW

File: test/test_actuator.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    assert not actuator.cmd_queue.empty()
+    assert actuator.cmd_queue.get() == "test_command"
+
+
+def test_get_command_with_command(actuator):
+    actuator.send_command(TestEnum.TEST_CMD)
+    command = actuator.get_command()
+    assert command == "test_command"
+    assert actuator.last_update is not None
+
```
Comment (cdimauro) [inline test/test_actuator.py:37]: assert actuator.cmd_queue.empty()

Comment (cdimauro) [reply test/test_actuator.py:37]: I don't know if it makes sense to check the queue status on all tests. I mean: before getting a value and after.

File: test/test_actuator_server.py
[the remark this answers is not in the document]
Comment (alessandropalla) [reply test/test_actuator_server.py:1]: I think so

File: test/test_actuator.py (hunk 1 above)
Comment (alessandropalla) [reply test/test_actuator.py:38]: done

**Pull request merged**